classlib/src/main/java/org/teavm/classlib/java/util/stream/intimpl/TStringCharsStream.java

```diff
@@ -21,10 +21,11 @@
 
 public class TStringCharsStream extends TSimpleIntStreamImpl {
     private final TString string;
-    private int index = 0;
+    private int index;
 
     public TStringCharsStream(TString string) {
         this.string = Objects.requireNonNull(string);
+        this.index = 0;
```

Comment:
You don't need this. Java spec guarantees to initialize int fields with 0

Reply:
I know, but explicitly stating that we start at index 0 seemed nicer.